code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -48,7 +49,8 @@ var/list/gaslist_cache = null
 	src.volume = volume
 	last_share = 0
 	fuel_burnt = 0
-
+	create_reagents(1000)
+	reagents.set_reacting(FALSE) // no

```

Comment:
Reagents datums aren't really supposed to stop existing after they start existing on an object. I can look into it, but no guarantees